src/status.rs

```diff
@@ -49,6 +49,7 @@ pub struct State {
 #[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
 pub struct Event {
     pub name: String,
+    #[serde(rename = "type")]
     pub _type: String,
```

Comment:
I opened an issue in the Style RFC repo since I couldn't find anything about it: https://github.com/rust-lang-nursery/fmt-rfcs/issues/84